go-controller/pkg/ovn/controller/apbroute/external_controller_policy.go

```diff
@@ -244,8 +247,12 @@ func (m *externalPolicyManager) applyPodConfig(pod *v1.Pod, existingPodConfig *p
 			gwsToAdd.InsertOverwrite(newGW)
 		}
 	}
+	klog.V(5).Infof("Adding GWs %+v to pod %s/%s", gwsToAdd, pod.Namespace, pod.Name)
```

Comment:
why do we need this log? only for dynamic gateways

Reply:
I want to keep it here for future debugging references, I use level 5 to keep it from showing up all the time.

Reply:
this will generate a lot of logs if we enable debug level, which we do sometimes in prod, so we still need to be thoughtful. Maybe we can put it under `if gwsToAdd.Len() > 0 {`?

Reply:
Well the idea I had was to show how many IPs were added to the pod before it went inside the conditional. Because adding it after the conditional would fail to catch the case where no gateway IPs were to be added. I will remove it if it can impact the level of logs.

Reply:
we can consider maybe printing something on a per-policy level, because this log will be seen for every pod, even if there is nothing to do, which may be a lot of logs

Reply:
As far as I understand, only pods that belong to a target namespace will be listed in the logs. It's possible that the log entries might be too numerous, so I removed the entry for now. 

I will leave it for future discussions if there is a need to have a similar log for debugging purposes due to numerous issues found in this particular area.